packages/itwin/reports-config-widget/src/widget/components/Reports.tsx

```diff
@@ -181,45 +254,52 @@ export const Reports = () => {
                       setSelectedReport(report);
                       setReportsView(ReportsView.REPORTSMAPPING);
                     }}
-                    button={
-                      <DropdownMenu
-                        menuItems={(close: () => void) => [
-                          <MenuItem
-                            key={0}
-                            onClick={() => {
-                              setSelectedReport(report);
-                              setReportsView(ReportsView.MODIFYING);
-                            }}
-                            icon={<SvgEdit />}
-                          >
-                            {ReportsConfigWidget.localization.getLocalizedString(
-                              "ReportsConfigWidget:Modify"
-                            )}
-                          </MenuItem>,
-                          <MenuItem
-                            key={1}
-                            onClick={() => {
-                              setSelectedReport(report);
-                              setShowDeleteModal(true);
-                              close();
-                            }}
-                            icon={<SvgDelete />}
+                    selected={selectedReports.some((r) => report.id === r.id)}
+                    onClick={(e) => onClickTile(e, report)}
+                    actionGroup={
+                      <div className="rcw-button-container">
+                        <ExtractionStatus
+                          state={extractionStates.get(report.id) ?? ExtractionStates.None}
+                          clearExtractionState={() => {
+                            extractionStates.delete(report.id);
+                            bulkExtractor.clearJob(report.id);
+                          }}
+                        >
+                          <DropdownMenu
```

Comment:
`DropdownMenu` should not be a child of `ExtractionStatus`. `actionGroup` should container `ExtractionStatus` OR `DropdownMenu`.

Reply:
`ExtractionStatus` moved to `ReportHorizontalTile`. `DropdownMenu` is now in `actionGroup` instead of child.